src/components/JbrowseLinear.tsx

```diff
@@ -21,19 +21,26 @@ import { createViewState, JBrowseLinearGenomeView } from '@jbrowse/react-linear-
 import { useEffect, useState } from 'react';
 import { defaultLinearOptions } from '../utils/linear/dynamic';
 import { getTracks } from './common';
-import { assembly } from '../utils/assembly';
-import { JbrowseFileInput } from './types';
+import { JbrowseFileInput, JbrowseLinearDefaultSession } from './types';
 
-export type LinearViewModel = ReturnType<typeof createViewState>;
+export type JbrowseLinearViewModel = ReturnType<typeof createViewState>;
+export type JbrowseLinearAssembly = JbrowseLinearViewModel['config']['assembly'];
+export type JbrowseLinearConfiguration = JbrowseLinearViewModel['config']['configuration'];
```

Comment:
this one needs a trigger warning for me 🤣 
I go "ooooffff"... every. single. time!